File: Gombang/Weekly Contest 412/3264_Final_Array_State_After_K_Multiplication_Operations_I.cs
public class Solution
{
	public int[] GetFinalState(int[] nums, int k, int multiplier)
	{
		for (int i = 0; i < k; i++)
		{
			int minIndex = 0;

			// 가장 작은 값의 인덱스를 찾는 과정 진행
			for (int j = 1; j < nums.Length; j++)
			{
				if (nums[j] < nums[minIndex])
					minIndex = j;
			}

			nums[minIndex] *= multiplier;
		}

		return nums;
	}
}

File: Seori/Weekly Contest 412/3264. Final Array State After K Multiplication Operations I.py
class Solution:
    def getFinalState(self, nums: List[int], k: int, multiplier: int) -> List[int]:
        # [1] 연산을 k번 반복한다. Repeat the operation for k times.
        for _ in range(k):
            # [2] 최소값의 인덱스를 찾아 multiplier를 곱한다. Find the index of the minimum value and multiply it by the multiplier.
            index_minV = nums.index(min(nums))
            nums[index_minV] *= multiplier
        return nums